VegetableSalad/VegetableSalad/Services/FindVegetable.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace VegetableSalad.Services
+{
+    internal class FindVegetable
+    {
+        public void FindTheMostCaloricVegetable(MakeSalad saladDish)
+        {
+            var mySalad = saladDish.MySalad();
+            double caloriesOfVegetable = (double)(mySalad[0].CaloriesPerGram) / (double)(100) * mySalad[0].Weight;
+            int index = 0;
+
+            foreach (var item in mySalad)
+            {
+                double calories = (double)(item.CaloriesPerGram) / (double)(100) * item.Weight;
```

Comment:
What's the difference between this  **double caloriesOfVegetable** and  **double calories**  variables